cmd/admin-top-locks.go

```diff
@@ -0,0 +1,152 @@
+/*
+ * Minio Client (C) 2019 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminTopLocksCmd = cli.Command{
+	Name:   "locks",
+	Usage:  "Get list of oldest locks on a Minio server/cluster.",
+	Before: setGlobalsFromContext,
+	Action: mainAdminTopLocks,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Get oldest locks on a Minio server/cluster.
+     $ {{.HelpName}} play/
+
+`,
+}
+
+// lockMessage struct to list lock information.
+type lockMessage struct {
+	Status string           `json:"status"`
+	Lock   madmin.LockEntry `json:"locks"`
+}
+
+func getTimeDiff(timeStamp time.Time) (string, string) {
+	loc, _ := time.LoadLocation("UTC")
+	now := time.Now().In(loc)
```

Comment:
it is sufficient to do this: time.Now().UTC()

Reply:
done